### DISCUSSION TITLE
Music production

### DISCUSSION THREAD

**Discussion by mrzapp** [OpenTournament/OpenTournament#216] [Game Features]
Hi!

@wail directed me here and recommended that I poked you about this, @mothershipl 

To reiterate, I am a game programmer/musician, and my main inspiration is Alexander Brandon, the composer for the original Unreal Tournament game. I feel like my style would fit this project, so I am offering my services.

[Here](https://jeppez.app/music) is a small taste of what I do.

Do you have composers on the project already? I'd love to collaborate or just contribute my own original tracks. I mainly enjoy composing, but I could also mix/master some existing tracks, if needed.

**Comment by bass3** [comment]
I am not @mothershipl but your productions have nice hooks but instrumentals are very poorly mixed, and sound very amateurish.
All 3 tracks have simliarities with UT99 tracks, regarding main hooks and overall composition, but they lack "newer", fresh sound, mixing is muddy and overall mastering is poor.
I would give you tips to search for fresh sounds and try to fix your mixing, the overall composition already is "ok" and catchy.

**Reply by mrzapp** [reply]
Thanks for the feedback!

I half agree. I am neither a professional mixing nor mastering engineer, and I definitely have a lot to learn. The "newer" sound you're referring to, though, where certain components of the mix pierce your ears, is just not something I like listening to, so I wouldn't want to aim for that aesthetic regardless. That may be a problem for this project's creative vision, I can see that. 

The sounds are synthesised using ZynAddSubFX, except for the drums and guitar.

I'd really appreciate tips on mixing and mastering. Whatcha got?
